index.md

```diff
@@ -4,12 +4,12 @@ layout: overview
 ---
 
 Helix is a Rackspace chartered effort to unify the look, feel, and
-functionality of our interfaces&mdash;one system composed of this Helix design
+functionality of our interfaces &mdash; one system composed of this Helix design
```

Comment:
Remove the spaces around the em dash. Em dashes are supposed to be flush to the words around them. See http://rackerlabs.github.io/docs-rackspace/style-guide/style/punctuation.html#dashes

Reply:
TIL I misuse the em dash daily =p. I'll revert this.

Reply:
Spaces removed in newest commit (coming very soon.)